### PR TITLE
Add SubscriptionRequest model

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: registrations/migrations/0002_subscriptionrequest.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('contact', models.CharField(max_length=36)),
+                ('version', models.IntegerField(default=1)),
+                ('messageset_id', models.IntegerField()),
+                ('next_sequence_number', models.IntegerField(default=1)),
+                ('lang', models.CharField(max_length=6)),
+                ('active', models.BooleanField(default=True)),
+                ('completed', models.BooleanField(default=False)),
+                ('schedule', models.IntegerField(default=1)),
+                ('process_status', models.IntegerField(default=0)),
```
Comment (imsickofmaps) [inline registrations/migrations/0002_subscriptionrequest.py]: Remove

File: registrations/models.py [hunk 2]
```diff
@@ -77,3 +77,28 @@ def registration_post_save(sender, instance, created, **kwargs):
...
+
+class SubscriptionRequest(models.Model):
+    """ A data model that maps to the Stagebased Store
+    Subscription model. Created after a successful Registration
+    validation.
+    """
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    contact = models.CharField(max_length=36, null=False, blank=False)
+    version = models.IntegerField(default=1)
```
Comment (imsickofmaps) [inline registrations/models.py]: Remove

File: registrations/models.py [hunk 3]
```diff
@@ -77,3 +77,28 @@ def registration_post_save(sender, instance, created, **kwargs):
...
+    messageset_id = models.IntegerField(null=False, blank=False)
+    next_sequence_number = models.IntegerField(default=1, null=False,
+                                               blank=False)
+    lang = models.CharField(max_length=6, null=False, blank=False)
+    active = models.BooleanField(default=True)
+    completed = models.BooleanField(default=False)
+    schedule = models.IntegerField(default=1)
+    process_status = models.IntegerField(default=0, null=False, blank=False)
```
Comment (imsickofmaps) [inline registrations/models.py]: Remove

File: registrations/models.py [hunk 4]
```diff
@@ -77,3 +77,28 @@ def registration_post_save(sender, instance, created, **kwargs):
...
+    """
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    contact = models.CharField(max_length=36, null=False, blank=False)
+    version = models.IntegerField(default=1)
+    messageset_id = models.IntegerField(null=False, blank=False)
+    next_sequence_number = models.IntegerField(default=1, null=False,
+                                               blank=False)
+    lang = models.CharField(max_length=6, null=False, blank=False)
+    active = models.BooleanField(default=True)
```
Comment (imsickofmaps) [inline registrations/models.py]: Remove

File: registrations/models.py [hunk 5]
```diff
@@ -77,3 +77,28 @@ def registration_post_save(sender, instance, created, **kwargs):
...
+    completed = models.BooleanField(default=False)
```
Comment (imsickofmaps) [inline registrations/models.py]: Remove

File: registrations/models.py [hunk 6]
```diff
@@ -77,3 +77,28 @@ def registration_post_save(sender, instance, created, **kwargs):
...
+    schedule = models.IntegerField(default=1)
+    process_status = models.IntegerField(default=0, null=False, blank=False)
+    metadata = JSONField(null=True, blank=True)
```
Comment (imsickofmaps) [inline registrations/models.py]: Remove

File: registrations/tasks.py [hunk 7]
```diff
@@ -77,7 +87,7 @@ class ValidateRegistration(Task):
     def check_field_values(self, fields, registration_data):
         failures = []
         for field in fields:
-            if field in ["contact", "registered_by"]:
+            if field in ["mother_id", "receiver_id", "registrar_id"]:
```
Comment (imsickofmaps) [inline registrations/tasks.py]: replace `registrar_id` with `operator_id`

**Comment by imsickofmaps** [comment]
minor name change, some deletions then :+1:

**Pull request merged**